### ISSUE TITLE
Key Error - Edge Version API Change

### ISSUE THREAD

**Issue by jdeath** [klept0/MS-Rewards-Farmer#232]
### Before submitting a bug report...

- [X] This bug wasn't already reported.
(I have checked every bug report on GitHub)

- [X] I've cleared the sessions folder.

### Branch

develop

### Commit

c269afb

### Describe the bug

Looks like edge API changed. Product is now product. Releases is now releases, and ProductVersion is not productVersion, Architecture is architecture

### Copy and paste your error

```
File "/app/src/userAgentGenerator.py", line 144, in <genexpr>
(product for product in data if product["Product"] == "Stable"),
~~~~~~~^^^^^^^^^^^
KeyError: 'Product'
```

### Screenshots

fixed now

### Value of dashboard variable

not needed

**Comment by jdeath** [comment]
Looks like they reverted the capitalization back to the way it was. Sorry. 

https://edgeupdates.microsoft.com/api/products

**Comment by cal4** [comment]
Yeah noticed that as well, reverted for now but will push in a bit

Strange

**Comment by Kyrela** [comment]
A solution would be testing both (e.g `data.get('Xxxx') or data['xxxx']`)

**Comment by Kyrela** [comment]
Hi @jdeath, this should be fixed. Do you still have this problem with the latest version of the `develop` branch?

**Comment by jdeath** [comment]
Yes fixed. Been working well. I keep dev branch refreshed every week or so.
